nucleus/src/etc/atomist-config/instructions.txt

```diff
@@ -0,0 +1,3 @@
+1. Copy the atomist registration output from the quantum mechanic/atomist client start up log into atomist-registration.json.
```

Comment:
Can we put these instructions into a `README.md` Markdown formatted file please.
E.g.

````markdown
# Atomist webhook configuration

This Python script parses the Atomist log output and generates a Spring compatible YAML file that can be used with Spring configuration.

## Usage

1. Copy the Atomist registration output from the Quantum Mechanic start up log into `atomist-registration.json`.
2. Run `main.py`:

```console
nucleus/src/etc/atomist-config $ python main.py
```

3. Output will be generated in `output.yml`. This YAML configuration can be placed in the `application.yml` file to configure the Atomist webhooks.
````

Reply:
@kbristow perhaps instead of adding the generated webhook URL's from `output.yml` to your `application.yml` manually and risk it being committed and pushed. We instead rename the `output.yml` generated file to `application-local.yml` and then when you run Gluon, you simply specify the `local` profile which will then load up the `application-local.yml` file automatically (will include `application.yml` because it's transitive...).
E.g.

```console
$ ./mvnw spring-boot:run --spring.profiles.active=local
```

Then we can add an entry to `.gitignore` for `application-local.yml`, preventing us accidentally committing.

Thoughts?

Reply:
If you agree, then also please add an entry in the Gluon README [Development setup section](https://github.com/absa-subatomic/gluon/blob/master/README.md#development-setup) to run with the `local` profile as per above.

Reply:
If I had known you could do that then I would have already done it :) will look into it

Reply:
No problem. See https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#boot-features-external-config-profile-specific-properties